src/components/TextField/TextField.tsx

```diff
@@ -383,6 +388,16 @@ export default class TextField extends React.PureComponent<Props, State> {
   private handleClick() {
     this.input.focus();
   }
+
+  @autobind
+  private handleButtonPress(onChange: Function) {
+    this.buttonPressTimer = setInterval(onChange, 200);
```

Comment:
`200` might be a bit long and I think we should consider adding a `setTimeout` first. That way when clicking and holding, there is an initial delay of maybe `500` then the number quickly increments maybe at an interval of `100`. Thoughts?

Reply:
☝️ I like this suggestion of gradually increasing the speed. Overall, this looks great!